src/profile-logic/profile-data.js

```diff
@@ -892,7 +905,7 @@ export function getTracingMarkers(thread: Thread): TracingMarker[] {
             dur: duration,
             name,
             data,
-            title: `${name} for ${duration.toFixed(2)}ms`,
+            title: getMarkerTitle(name, data),
```

Comment:
I thought I would be able to use it for the MarkersView and that's why I did it here -- in the end the MarkersView doesn't use this code at all so meh. I still think it's a good idea to have it here, but please tell me if you think otherwise.